drivers/SmartThings/philips-hue/src/handlers/refresh_handlers.lua

```diff
@@ -0,0 +1,256 @@
+local cosock = require "cosock"
+local log = require "log"
+local st_utils = require "st.utils"
+
+local Fields = require "fields"
+local HueDeviceTypes = require "hue_device_types"
+local utils = require "utils"
+
+local M = {}
+
+local device_type_refresh_handlers_map = {}
+
+---@param driver HueDriver
+---@param bridge_device HueBridgeDevice
+function M.do_refresh_all_for_bridge(driver, bridge_device)
+  cosock.spawn(
+    function()
+      local child_devices = bridge_device:get_child_list() --[=[@as HueChildDevice[]]=]
+
+      if not bridge_device:get_field(Fields._INIT) then
+        log.warn("Bridge for lights not yet initialized, can't refresh yet.")
+        return
+      end
+
+      local hue_api = bridge_device:get_field(Fields.BRIDGE_API) --[[@as PhilipsHueApi]]
```

Comment:
you've got `--[=[` above and `--[[` here and these annotations seem to have broken github's syntax highlighting so just double-checking this is all correct.